src/middleware/options/ChaosHandlerOptions.ts

```diff
@@ -48,20 +48,30 @@ export class ChaosHandlerOptions implements MiddlewareOptions {
 	 */
 	public chaosPercentage: number;
 
+	/**
+	 * The response body to be returned in the response
+	 *
+	 * @public
+	 */
+	public responseBody: any;
+
 	/**
 	 * @public
 	 * @constructor
 	 * To create an instance of Testing Handler Options
 	 * @param {ChaosStrategy} ChaosStrategy - Specifies the startegy used for the Testing Handler -> RAMDOM/MANUAL
-	 * @param {number?} statusCode - The Message to be returned in the response
-	 * @param {string} - The Message to be returned in the response
+	 * @param {number?} statusCode - The statusCode to be returned in the response
+	 * @param {string} statusMessage - The Message to be returned in the response
+	 * @param {number?} chaosPercentage - The percentage of randomness/chaos in the handler
+	 * @param {any} responseBody - The response body to be returned in the response
```

Comment:
Indicate that responseBody is optional

`@param {any?}`

Reply:
Done